vassal-app/src/main/java/VASSAL/build/module/properties/EnumeratedPropertyPrompt.java

```diff
@@ -54,6 +54,10 @@ public EnumeratedPropertyPrompt(DialogParent dialogParent, String prompt, String
     this.propertySource = propertySource;
   }
 
+  public Expression[] getValueExpressions() {
+    return valueExpressions;
+  }
+
```

Comment:
?

Reply:
Took this one out and it turns out to be needed online 288 of ChangePropertyButton.java, in my new getExpressionList() thing, so I guess I added it for that.